raiden/transfer/architecture.py

```diff
@@ -164,12 +164,17 @@ class Event:
     __slots__ = ()
 
 
-class SendMessageEvent(Event):
+class EventWithLogTime(Event):
```

Comment:
I see the point of creating the marker... but it would have made more sense if we:
1. Define common attributes, i.e `log_time` and reuse this in child classes, OR
2. In [your WAL change](https://github.com/raiden-network/raiden/pull/2381/files#diff-34fcdfdd66b47942cc02bf2a12fa4925R173) we check if `isinstance(event, EventWithLogTime)` and then set the attribute value.
However, the marker in it's current form doesn't do anything which means introducing it is not necessary. `Event` could still serve as the base class if we're just going to set `log_time` for all events.